### PR TITLE
adding in smol raptor borgs from bubber

### PR DESCRIPTION
**Pull request by Asher-42**
adding in smol raptor borgs from bubberstation!

## Changelog

:cl:
add: Added in smol raptor borgs from bubber station!
/:cl:

dreamseeker_YHr0oJS6Uu

### CODE REVIEW

**Comment by aKromatopzia** [comment]
hi artist here just note that the _gen sprites are slightly outdated - but they're debug sprites so you shouldn't see them anyways; so if you want to make new sprites (for example bellies) just make sure not to use those

**Comment by Asher-42** [comment]
> hi artist here just note that the _gen sprites are slightly outdated - but they're debug sprites so you shouldn't see them anyways; so if you want to make new sprites (for example bellies) just make sure not to use those

gotcha! il add you in a file for credit too, not sure if im blind but i didnt see a source file when i copied over

File: modular_chomp/code/modules/mob/living/silicon/robot/sprites/mining.dm [hunk 1]
```diff
@@ -110,3 +110,26 @@
...
+/datum/robot_sprite/dogborg/mining/smolraptorcargosprite
+	sprite_icon = 'modular_chomp/icons/mob/smallraptors/smolraptor_car.dmi'
+
+
+/datum/robot_sprite/dogborg/mining/smolraptorminersprite/smolraptorminer
+	name = "Small Raptor"
+	sprite_icon_state = "smolraptor"
+	has_eye_light_sprites = TRUE
+	has_custom_open_sprites = TRUE
+	has_vore_belly_sprites = FALSE
```
Comment (Kashargul) [inline modular_chomp/code/modules/mob/living/silicon/robot/sprites/mining.dm:126]: Why is this set to true here?

Comment (Asher-42) [reply modular_chomp/code/modules/mob/living/silicon/robot/sprites/mining.dm:126]: no idea. odd! i copy pasted everything. one moment.

Comment (Asher-42) [reply modular_chomp/code/modules/mob/living/silicon/robot/sprites/mining.dm:126]: the heck, its false for me

**Pull request merged**